plugins/git-checkout-interactive/git-checkout-interactive.plugin.zsh

```diff
@@ -0,0 +1,57 @@
+#######################################
+#       git checkout interactive      #
+#######################################
+
+function git-checkout-interactive() {
+  local ITEMS_TO_SHOW=10
```

Comment:
This doesn't necessarily need to be changed (it's not a problem, nor something I consider a significant problem), but why have a function-local constant as a variable if it can't be configured in some way?